phpcs.xml

```diff
@@ -1,54 +1,26 @@
-<?xml version="1.0" ?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
   Based on Examples for Developers phpcs.xml.dist. Modified for use for Drupal
   projects. See https://git.drupalcode.org/project/examples/blob/HEAD/phpcs.xml.dist
   for source.
 -->
 <ruleset name="AxelerantDrupal">
   <description>Axelerant baseline rules for PHP CodeSniffer</description>
-
-  <!--Exclude folders used by common frontend tools.-->
-  <exclude-pattern>*/bower_components/*</exclude-pattern>
-  <exclude-pattern>*/node_modules/*</exclude-pattern>
-  <!--Exclude third party code.-->
-  <exclude-pattern>./assets/vendor/*</exclude-pattern>
-
   <file>.</file>
-
-  <!-- PHPCS Arguments -->
-  <arg name="extensions" value="inc,yml,install,module,php,profile,test,theme" />
-  <arg name="colors" />
-
-  <!--Allow global variables in settings file.-->
-  <rule ref="DrupalPractice.CodeAnalysis.VariableAnalysis.UndefinedVariable">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
-  </rule>
-
-  <!--Allow section separators in settings.php file.-->
-  <rule ref="DrupalPractice.Commenting.CommentEmptyLine.SpacingAfter">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
-  </rule>
-  <rule ref="Drupal.Commenting.InlineComment.InvalidEndChar">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
-  </rule>
-  <rule ref="Drupal.Commenting.InlineComment.NoSpaceBefore">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
-  </rule>
-  <rule ref="Drupal.Commenting.InlineComment.SpacingAfter">
-    <exclude-pattern>settings(\.[a-zA-Z0-9]+)?\.php</exclude-pattern>
-  </rule>
-
+  <!-- Only include specific sniffs that pass. This ensures that, if new sniffs are added, HEAD does not fail.-->
   <!-- Drupal sniffs -->
+  <rule ref="Drupal"/>
   <rule ref="Drupal.Arrays.Array">
     <!-- Sniff for these errors: CommaLastItem -->
+    <exclude name="Drupal.Arrays.Array.ArrayClosingIndentation"/>
+    <exclude name="Drupal.Arrays.Array.ArrayIndentation"/>
     <exclude name="Drupal.Arrays.Array.LongLineDeclaration"/>
   </rule>
   <rule ref="Drupal.Classes.ClassCreateInstance"/>
   <rule ref="Drupal.Classes.ClassDeclaration"/>
   <rule ref="Drupal.Classes.FullyQualifiedNamespace"/>
   <rule ref="Drupal.Classes.InterfaceName"/>
   <rule ref="Drupal.Classes.UnusedUseStatement"/>
-  <rule ref="Drupal.Classes.UseGlobalClass"/>
```

Comment:
Any specific reason for removing this? This checks the following.

You cannot `use` the use statement for non-namespaced classes; instead, you must reference them with a leading `\`, e.g., `\DateTimeImmutable` instead of `use DateTimeImmutable;`.